osu.Game/Skinning/LegacyManiaSkinConfiguration.cs

```diff
@@ -29,6 +29,8 @@ public class LegacyManiaSkinConfiguration : IHasCustomColours
 
         public Dictionary<string, string> ImageLookups = new Dictionary<string, string>();
 
+        public float WidthForNoteHeightScale;
```

Comment:
I added this because I thought it was required but it turns out it wasn't. Will leave it in place because I'm sure we'll have to use it in the future to support some other skins.